test/tools/cli/IonProcess.java

```diff
@@ -544,14 +502,28 @@ private static Event ionStreamToEvent(IonReader ionReader) {
                     event.setDepth(ionReader.intValue());
                     break;
             }
-        }
 
+        }
+        //compare text value and binary value
+        if (textValue != null && binaryValue != null) {
+            IonSystem ionSystem = IonSystemBuilder.standard().build();
```

Comment:
This can be a constant. No need to reinstantiate it every time.

Reply:
Changed (IonProcess.java 47)